custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -195,15 +198,31 @@ def create_diagnosis_and_notification_payload(cls, episode_case):
     def create_ayush_referral_payload(cls, episode_case):
         episode_case_properties = episode_case.dynamic_case_properties()
 
-        location = cls._get_location(
-            episode_case_properties.get("registered_by"),
-            field_name="registered_by",
-            related_case_type="episode",
-            related_case_id=episode_case.case_id,
-        )
-        if not location.user_id:
-            raise NikshayLocationNotFound(
-                "Location {} does not have a virtual location user".format(location.location_id))
+        registered_by_location = episode_case_properties.get("registered_by")
```

Comment:
If it was only test cases that were failing this test, then I'm happy to leave this as it was, as users are only on the newest version of the app, which sets this property correctly.